apps/verticalhome/js/clock.js

```diff
@@ -0,0 +1,86 @@
+/**
+  Copyright 2012, Mozilla Foundation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
```

Comment:
We don't have these comments at the top of the other new files, so let's not start adding them now.  The fact that the files are being submitted as pull requests to a repo with a LICENSE file in it will have to be good enough to indicate the license